gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/rest/api/service/impl/MetadataServiceTest.java

```diff
@@ -118,4 +125,93 @@ public void checkMetadataFormat_badDateFormat() {
     public void checkMetadataFormat_Date() {
         metadataService.checkMetadataFormat(GraviteeContext.getExecutionContext(), MetadataFormat.DATE, "2015-02-24");
     }
+
+    @Test
+    public void createWithReference() throws TechnicalException {
```

Comment:
In the existing createMetadata method the auditService is called. In the new method introduced by this PR audit isn't call. If it is done with intention I suggestion to put an assertion here to check that the audit service is never call. wdyt ?

Reply:
Yes, done.